demisto_sdk/commands/pre_commit/.pre-commit-config_template.yaml

```
@@ -0,0 +1,79 @@
+exclude: .devcontainer/|.vscode|Pipfile.lock|.gitlab/ci/|.circleci/
+
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.4.0
+    hooks:
+      - id: trailing-whitespace
+      - id: check-json
+      - id: check-yaml
+      - id: check-ast
+      - id: check-merge-conflict
+      - id: debug-statements
+        language_version: python3
+      - id: name-tests-test
+        files: .+_test.py$
+
+  - repo: https://github.com/charliermarsh/ruff-pre-commit
+    rev: "v0.0.257"
```

```suggestion
    rev: "v0.0.258"
```
nobody can keep up :)